### PR TITLE
docs: add section regarding ssh_domains auto-populate and its use

### PR DESCRIPTION
**Pull request by netboy3**

### CODE REVIEW

File: docs/multiplexing.md [hunk 1]
```diff
@@ -64,6 +64,16 @@ the wezterm multiplexer daemon on the remote host and connect to
 it via a unix domain socket using a similar mechanism to that
 described in the *Unix Domains* section below.
 
+{{since('20230408-112425-69ae8472')}}
+
+Ssh_domains now auto-populate from your ~/.ssh/config file. Each populated host will have both a plain SSH and a multiplexing SSH domain. Plain SSH hosts are defined with a `SSH:` prefix to their name and multiplexing hosts are defined with a prefix `SSHMUX:`. For example, to connect to a host named `my.server` in your ~/.ssh/config using a multiplexing domain, run:
```
Comment (wez) [inline docs/multiplexing.md:69]:
```suggestion
Ssh_domains now auto-populate from your ~/.ssh/config file. Each populated host will have both a plain SSH and a multiplexing SSH domain. Plain SSH hosts are defined with a `SSH:` prefix to their name and multiplexing hosts are defined with a prefix `SSHMUX:`. For example, to connect to a host named `my.server` in your `~/.ssh/config` using a multiplexing domain, run:
```

File: docs/multiplexing.md [hunk 2]
```diff
@@ -64,6 +64,16 @@ the wezterm multiplexer daemon on the remote host and connect to
...
+
+```console
+$ wezterm connect SSHMUX:my.server
```
Comment (wez) [inline docs/multiplexing.md:72]:
```suggestion
$ wezterm connect SSHMUX:my.server
# or to spawn into a new tab in an existing wezterm gui instance:
$ wezterm cli spawn --domain-name SSHMUX:my.server
```

File: docs/multiplexing.md [hunk 3]
```diff
@@ -64,6 +64,16 @@ the wezterm multiplexer daemon on the remote host and connect to
...
+```
+
+To customize this functionality, You can use [wezterm.default_ssh_domains()](config/lua/wezterm/default_ssh_domains.md)
```
Comment (wez) [inline docs/multiplexing.md:75]:
```suggestion
To customize this functionality, see the example for [wezterm.default_ssh_domains()](config/lua/wezterm/default_ssh_domains.md)
```

**Review by wez — COMMENTED** [review]
Thanks!

File: docs/multiplexing.md (hunk 1 above)
Comment (netboy3) [reply docs/multiplexing.md]: Pushed 9df9ea7e

File: docs/multiplexing.md (hunk 2 above)
Comment (netboy3) [reply docs/multiplexing.md:72]: Pushed 9df9ea7e

File: docs/multiplexing.md (hunk 3 above)
Comment (netboy3) [reply docs/multiplexing.md]: Pushed 9df9ea7e

**Comment by wez** [comment]
Thank you!

**Pull request merged**